source/core/ut_output_pipe_helper.pks

```diff
@@ -1,5 +1,7 @@
 create or replace package ut_output_pipe_helper is
 
+  gc_enabled constant boolean := $if $$ut_pipe_enable $then TRUE; $else FALSE; $end
```

Comment:
we it was going to eb needed to know if dbms_pipe was enabled or not.. but since we're takign out teh $cc then this can go too.. yes.